cfgen/cfgen

```diff
@@ -2004,9 +2014,16 @@ def aux_pool_list(pool_desc: Dict[str, Any], m0conf: Dict[Oid, Any]) -> None:
     # Generate the node combinations considering node failures as
     # allowed by node tolerance values
     temp_pool_nodes = []
-    for nodes in pool['disk_refs']:
-        temp_pool_nodes += [
-            nodes['node']]
+
+    disk_refs = pool.get('disk_refs')
```

Comment:
` aux_pool_list()` has grown big and we are compiling `disk_refs` at-least twice in this patch. Please create a function to get disk refs from pool_desc. Accept pool_desc and a parameter.

Reply:
Done